articles/deciding-between-cloud-spanner-and-microsoft-cosmos-db/index.md

```diff
@@ -0,0 +1,55 @@
+Creating a globally distributed database would traditionally require a lot of time and effort. You had to host the database in an own data center and use your own connections. Today&#39;s advancement in cloud computing and platforms as a service (PaaS) has made it easier to create globally distributed and scalable databases. Particularly, the use of SQL-based databases has become more popular in the recent past. These databases provide you with a customized experience of sorting and processing large amounts of raw data. Many database providers exist, and they work on different nodes of the SQL database engines, including [Microsoft Azure Cosmos DB](https://docs.microsoft.com/en-us/azure/cosmos-db/introduction) and [Google Cloud Spanner](https://cloud.google.com/spanner).
```

Comment:
You would ou had to host the database in an own data center and use your own connections->have to host the database in a personal data center bearing the entire cost.